package.json

```diff
@@ -46,7 +46,7 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "antd": "^4.22.8",
+    "antd": "5.9.0",
```

Comment:
@Mil05h Using hard versions here will make bundle pull in multiple antd versions if the main project package.json is updated. We should set minimum versions in this package so using "^5.9.0" instead is the way to go, then it will use main package.json antd version instead as long as it is greater than 5.x.x.

Reply:
Makes sense. Thanks for the explanation!